### DISCUSSION TITLE
Implementing beatmap modding

### DISCUSSION THREAD

**Discussion by Ianlucht** [ppy/osu#16725] [Ideas] [answered]
I think it could be an interesting Idea to implement beatmap modding directly into the editor. Doing this could streamline the process for mappers and modders alike, being able to make comments on parts of the map in real time inside the editor would be far easier than needing to move to the osu! website to do it and doing this would also lower the bar for those trying to enter the space and make the prospect of modding and editing less daunting to the newer participants of beatmap creation and modding.

My proposals for what this could look like: 

-  An extra tab in the editor for modders to make comments at any point in the map, as well as specify between comments and problems related to the map.
- A message box on the side or top of the editor for the beatmap creator to see what modders have to say about the section and a resolve button for the mod

**Comment by smoogipoo** [comment, marked as answer]
I think this falls under https://github.com/ppy/osu/issues/5379, though we didn't have the current beatmap modding page when that issue was made. We still don't have in-client designs for this, though.
